### PR TITLE
Added bundler config for cocoapods

### PR DESCRIPTION
**Pull request by AlexYanski**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 source "https://rubygems.org"
 
-gem "fastlane"
+gem 'fastlane'
+gem 'cocoapods', '~> 1.7.5'
```
Comment (pkondrashkov) [inline Gemfile]: Let's maybe update this version to atleast 1.8?

File: Gemfile [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 source "https://rubygems.org"
 
-gem "fastlane"
+gem 'fastlane'
```
Comment (pkondrashkov) [inline Gemfile]: I think we should also fix this one to some version

File: README.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ bundle install
 5. Install pods:
 
 ```bash
-pod install
+bundle exec pod install
```
Comment (pkondrashkov) [inline README.md]: I dont see the step where we install bundler itsels and its gems.

Comment (AlexYanski) [reply README.md]: Fixed

File: Gemfile (hunk 2 above)
Comment (AlexYanski) [reply Gemfile]: done

File: Gemfile (hunk 1 above)
Comment (AlexYanski) [reply Gemfile]: done

File: README.md (hunk 3 above)
Comment (pkondrashkov) [reply README.md]: Still don't see instructions how to install bundler and it's gems.

Comment (AlexYanski) [reply README.md]: It's a little higher, at number 5. I fixed 'Install pods' number, now it's 6

Comment (pkondrashkov) [reply README.md]: Oh right. But will the bundle install work? I think you have to specify the path where to install the gems.

Comment (AlexYanski) [reply README.md]: So what the path should be?

File: Gemfile [hunk 4]
```diff
@@ -2,4 +2,5 @@
 
 source "https://rubygems.org"
 
-gem "fastlane"
+gem 'fastlane', '~> 2.144'
+gem 'cocoapods', '~> 1.8'
```
Comment (pkondrashkov) [inline Gemfile:6]: Add new line after gem 'cocoapods', '~> 1.8' please

**Pull request merged**